### ISSUE TITLE
[Feature request] Error message when importing a product twice

### ISSUE THREAD

**Issue by Hocdoc** [akeneo/pim-community-dev#4298]
One of our users added the same product update twice in a `product.csv`, which produce a complicate error message (see below) during the import. It would be great, if you can give a nicer error message for the end user.

Steps to reproduce it with Akeneo 1.5.1, http://demo.akeneo.com/ :

Create a minimal `product.csv` like this:

```
sku;enabled
10699783;0
10699783;0
```

When you import this file you will get this only-for-developer-eyes-error message:

```
PRODUKT-IMPORT  An exception occurred while executing 'INSERT INTO akeneo_file_storage_file_info (file_key, original_filename, mime_type, size, extension, hash, storage) VALUES (?, ?, ?, ?, ?, ?, ?)' with params ["a\/c\/f\/e\/acfe1a107a4079a3e36936e95cdfde339900124b_10699783_840.jpg", "10699783-840.jpg", "image\/jpeg", 6451, "jpg", "b4a12a4dfc68411a41fc52fedb17f32fd7645d99", "catalogStorage"]: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'a/c/f/e/acfe1a107a4079a3e36936e95cdfde339900124b_10699783_840.jp' for key 'UNIQ_F19B3719A5D32530'
```

**Comment by willy-ahva** [comment]
Hi @Hocdoc !

Thank you for the feedback !
We know that we have some efforts to do on the error messages topic. This kind of feedback help us a lot :) I'll add this enhancement in our internal tracker.

Thanks again !
Willy

**Comment by a2xchip** [comment]
@willy-ahva Is it really error?

**Comment by Hocdoc** [comment]
Fixed in Akeneo 1.7

**Comment by peleq-smile** [comment]
Hi,

Is it possible to get the commit fixing this issue please ?

Thkx,
Perrine
